src/package/abi/Microsoft.Windows.AbiWinRT.targets

```diff
@@ -6,8 +6,493 @@ Copyright (C) Microsoft Corporation. All rights reserved.
+        <CAExcludePath>$(GeneratedFilesDir);$(CAExcludePath)</CAExcludePath>
+
+        <PrepareForBuildDependsOn>
+            $(PrepareForBuildDependsOn);
+            AbiWinRTVerifyKitVersion;
+        </PrepareForBuildDependsOn>
+        <!-- Note: Before* targets run before Compute* targets. -->
+        <ComputeMidlInputsTargets>
+            $(ComputeMidlInputsTargets);AbiWinRTSetMidlReferences;
+        </ComputeMidlInputsTargets>
+        <AfterMidlTargets>
+            $(AfterMidlTargets);
+            GetAbiWinRTMdMergeInputs;
+            AbiWinRTMergeProjectWinMDInputs;
+            GetResolvedWinMD;
+            AbiWinRTCopyWinMDToOutputDirectory;
+        </AfterMidlTargets>
+        <ResolveAssemblyReferencesDependsOn>
+            $(ResolveAssemblyReferencesDependsOn);GetAbiWinRTProjectWinMDReferences;AbiWinRTMarkStaticLibrariesPrivate;
+        </ResolveAssemblyReferencesDependsOn>
+        <!-- Note: Before* targets run before Compute* targets. -->
+        <BeforeClCompileTargets>
+            $(BeforeClCompileTargets);AbiWinRTMakeProjections;
+        </BeforeClCompileTargets>
+        <CleanDependsOn>
+            $(CleanDependsOn);AbiWinRTClean
+        </CleanDependsOn>
+
+    </PropertyGroup>
+
+    <!-- For a static library we don't want the winmd/lib/pdb to be packaged -->
+    <PropertyGroup Condition="'$(ConfigurationType)' == 'StaticLibrary'">
+        <IncludeCopyWinMDArtifactsOutputGroup>false</IncludeCopyWinMDArtifactsOutputGroup>
+    </PropertyGroup>
+
+    <Target Name="AbiWinRTVerifyKitVersion" Condition="'$(AbiWinRTOverrideSDKReferences)' != 'true'">
+        <PropertyGroup>
+            <_AbiWinRT_RS4OrGreater>false</_AbiWinRT_RS4OrGreater>
+            <_AbiWinRT_RS4OrGreater Condition="'$(TargetPlatformVersion)' &gt;= '10.0.17134.0'">true</_AbiWinRT_RS4OrGreater>
+        </PropertyGroup>
+        <VCMessage Code="MSB8036" Type="Error" Arguments="10.0.17134.0 (or later)" Condition="$(_AbiWinRT_RS4OrGreater) != 'true'" />
+    </Target>
+
+    <Target Name="AbiWinRTClean">
+        <ItemGroup>
+            <_FilesToDelete Remove="@(_FilesToDelete)"/>
+            <_FilesToDelete Include="$(GeneratedFilesDir)**"/>
+            <_FilesToDelete Include="$(OutDir)*.winmd"/>
+            <_FilesToDelete Include="$(IntDir)*.winmd"/>
+            <_FilesToDelete Include="$(IntDir)*.idl"/>
+            <_FilesToDelete Include="$(IntDir)*.rsp"/>
+            <_FilesToDelete Include="$(AbiWinRTMergedDir)**"/>
+            <_FilesToDelete Include="$(AbiWinRTUnmergedDir)**"/>
+        </ItemGroup>
+        <Delete Files="@(_FilesToDelete)"/>
+    </Target>
+
+    <!-- Target used only to evaluate AbiWinRTGenerateWindowsMetadata if it doesn't already have a value -->
+    <Target Name="AbiWinRTComputeGenerateWindowsMetadata"
+            DependsOnTargets="GetAbiWinRTProjectWinMDReferences;$(AbiWinRTComputeGenerateWindowsMetadataDependsOn)">
+
+        <PropertyGroup>
```

Comment:
this nuget package is "consume only" - referenced by clients of winmds - all static lib concerns should go